util/keyfile/src/error.rs

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! This module contains the error type which methods of the `keyfile` crate can
+//! generate.
+
+use crate::mnemonic_acct::Error as MnemonicAccountError;
+use displaydoc::Display;
+use mc_account_keys::Error as AccountKeyError;
+use mc_account_keys_slip10::Error as Slip10Error;
+use mc_util_serial::DecodeError as MCUtilSerialDecodeError;
```

Comment:
This is just a re-export of `prost::DecodeError`, so let's remove it

Reply:
Micro-nit: rust tends to use strict CamelCase names for types (e.g. `JsonError` and `IoError` below).